.env.test

```diff
@@ -1,3 +1,4 @@
+export ENVIRONMENT="test"
```

Comment:
can we have only one env file as both files have the same variables?

Reply:
removed the not needed env file.